great-idea/css/services.css

```diff
@@ -84,3 +85,10 @@ main section a:hover {
 .facts .left ol li {
   margin-bottom: 10px;
 }
+
```

Comment:
Great job organizing  your css. It's good practice for what we will be getting into in the next couple weeks with preprocessing.